templates/activate-mongodb-atlas-resources.yaml

```
@@ -0,0 +1,155 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Resource activation. (qs-1rkorhec7)
```

Can't reuse template IDs. Replace with `qs-1tq8f0hon`.
```suggestion
Description: MongoDB Resource activation. (qs-1tq8f0hon)
```